client/project.go

```diff
@@ -36,6 +42,22 @@ type Project struct {
 	DateCreated  int    `json:"date_created" model:"date_created" mapstructure:"date_created"`
 	DateModified int    `json:"date_modified" model:"date_modified" mapstructure:"date_modified"`
 	Status       string `model:"status" mapstructure:"status"`
+	SettingsData struct {
+		TimeFormat string `json:"time_format" mapstructure:"time_format"`
+		Timezone   string `json:"timezone" mapstructure:"timezone"`
+	} `json:"settings_data" mapstructure:"settings_data"`
+}
+
+// FlattenedProject is needed for data source
```

Comment:
Would it simplify things if we removed the nesting in the Project schema update?

Reply:
no need to change it, it's only needed for project data source.

Reply:
It's just modelling the API response? Would this separate struct be still required if the Project schema didn't have a fancy format?

Reply:
I don't think we can change it, it's public API

Reply:
I do not mean changing the API, just the Terraform schema representation.

Reply:
in that case it won't simplify on my end at all :)